src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -67,7 +67,8 @@
     #set ($discard = $xwiki.ssx.use('Diagram.DiagramViewSheet'))
     #set ($discard = $xwiki.jsx.use('Diagram.DiagramViewSheet'))
     {{html clean="false"}}
-    &lt;div class="diagram loading" data-model="$escapetool.xml($tdoc.content)"&gt;&lt;/div&gt;
+    ## Check if the query contains the parameter for getting the diagram from URL.
+    &lt;div class="diagram loading" #if ($request.export == '') data-model="$escapetool.xml($tdoc.content)" #end&gt;&lt;/div&gt;
```

Comment:
Better use the *silent* notation here also (so that ``null`` is handled as the empty string).
```suggestion
    &lt;div class="diagram loading" #if ("$!request.export" == '') data-model="$escapetool.xml($tdoc.content)" #end&gt;&lt;/div&gt;
```